gnomad_qc/v3/resources/sample_qc.py

```diff
@@ -166,6 +166,16 @@ def filtered_subpop_qc_mt(pop: str, version: str = CURRENT_VERSION) -> str:
     return f"{get_sample_qc_root(version)}/subpop_analysis/gnomad_v{version}_filtered_subpop_qc_mt.{pop}.mt"
 
 
+def assigned_subpops_path(pop: str, version: str = CURRENT_VERSION) -> str:
+    """
+    Path to Table of inferred sample subpopulations
+    :param pop: Population in which subpops have been assgined
+    :param version: Version of assigned subpops path to return
+    :return: String path to Table of assigned subpops within population specified by 'pop' parameter 
+    """
+    return f"{get_sample_qc_root(version)}/subpop_analysis/gnomad_v{version}_assigned_subpops.{pop}.ht"
```

Comment:
Should we make this a VersionedTableResource?

Reply:
i was going to copy what was done for "pop", but put here instead because of the extra pop argument